extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -139,5 +313,236 @@ public void registerSwaggerUiServletExtension(SwaggerUiRecorder recorder,
          */
         @ConfigItem(defaultValue = "true")
         boolean enable;
+
+        /**
+         * The urls that will be included as options. By default the OpenAPI path will be used.
+         * Here you can override that and supply multiple urls that will appear in the TopBar plugin.
+         * Format is name=url,name=url (if name is omitted, url will be use as name)
+         */
+        @ConfigItem
+        Optional<String[]> urls;
+
+        /**
+         * If urls option is used, this will be the name of the default selection.
+         */
+        @ConfigItem(name = "urlsPrimaryName")
```

Comment:
The default is camel-cased keys in application.properties/yaml. I think these properties should follow this pattern too

Reply:
I tried to keep it the same as : https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/ . I think that is maybe better ?

Reply:
Maybe that works for copy-pasting, but I'm on the fence about this TBH.

Reply:
No, we need to be consistent with the Quarkus way. Just remove the `name` everywhere.

This will be auto-documented in the doc (if the configuration reference has been added to the doc).

Reply:
If we later want to support the external config file from swagger-ui, names in the config will then be different. If we are ok with that, then I am happy. Personally, I used the swagger-ui documentation, and got caught a few times with the camelCase to camel-case in Quarkus, but might just be me.

Reply:
> If we later want to support the external config file from swagger-ui, names in the config will then be different.

Of course, `swagger-config.yaml` is totally different from `application.yaml`, that's not a problem.

IMHO consistency is key here, so +1 to remove the `name` everywhere

Reply:
I agree that consistency is key. (Except I believe it should be consistent with Swagger-UI :) ). But I'll change as suggested. What would be cool is if we could give a config an alias - so both is supported.